src/main/java/hudson/plugins/git/UserRemoteConfig.java

```diff
@@ -75,19 +80,22 @@ public String toString() {
     public static class DescriptorImpl extends Descriptor<UserRemoteConfig> {
 
         public ListBoxModel doFillCredentialsIdItems(@AncestorInPath Item project,
```

Comment:
I'm sorry, but I don't understand the context for use of Restricted.  Is that an annotation that is to be added to new API calls?  

What are the common patterns for use of that annotation?